charts/openbao/Chart.yaml

```diff
@@ -3,7 +3,7 @@
 
 apiVersion: v2
 name: openbao
-version: 0.15.0
+version: 0.16.0
```

Comment:
as we don't have "new feature" per se in the chart I would abstain from bumping minor. Can you do patchfix?